settings.gradle.kts

```
@@ -1,3 +1,7 @@
-include("android")
-include("coroutines")
-include("stdlib")
+include(
+    "android",
+    "coroutines",
+    "stdlib"
+)
```

```suggestion
include("android")
include("coroutines")
include("stdlib")
```
😐 😐 😐 😐 

It's just one less stupid diff conflict.